utils/i18nUtils.js

```diff
@@ -0,0 +1,97 @@
+import i18next from "i18next";
+import { initReactI18next } from "react-i18next";
+import resourcesToBackend from "i18next-resources-to-backend";
+import { match } from "@formatjs/intl-localematcher";
+import { shouldPolyfill } from "@formatjs/intl-locale/should-polyfill";
+import {
+  DEFAULT_LOCALE as DEFAULT_POLARIS_LOCALE,
+  SUPPORTED_LOCALES as SUPPORTED_POLARIS_LOCALES,
+} from "@shopify/polaris";
+
+// Update this to the default locale for the app
+const DEFAULT_LOCALE = "en";
+// Update this to the supported locales contained in the `locales` folder
+const SUPPORTED_APP_LOCALES = ["en", "fr"];
```

Comment:
Nit: Should these be made consistent?

Either:

```suggestion
const DEFAULT_LOCALE = "en";
// Update this to the supported locales contained in the `locales` folder
const SUPPORTED_LOCALES = ["en", "fr"];
```

or

```suggestion
const DEFAULT_APP_LOCALE = "en";
// Update this to the supported locales contained in the `locales` folder
const SUPPORTED_APP_LOCALES = ["en", "fr"];
```

Reply:
Done, thanks!